test/login.test.js

```diff
@@ -0,0 +1,132 @@
+const request = require('supertest');
+const app = require('../api/server').app;
+const assert = require('assert');
+const db = require('../api/models');
+const config = require('../api/config');
+const clone = require('clone');
+const fs = require('fs');
+
+const errorMessage = "Error: This is a top-10 common password - Add another word or two. Uncommon words are better.. Please choose a stronger password and try again.";
+describe('Signup API', () => {
+  before((done) => {
+    // Connect to test database
+    db.init(() => {
+      // use config.mongodb as test database and change it if env is not test
+      const dbString = process.env.NODE_ENV == "test" ? config.mongodb_test : config.mongodb;
```

Comment:
@anibalsolon Wanted to confirm if this is what you expected here 

https://github.com/brainlife/auth/pull/43#discussion_r1171982146

Reply:
Yes, my comment was a bit convoluted, let me improve that.

Configurations should be a single point of truth, in which (almost) independently of the environment you are working on, the configuration is guaranteed to have a valid value. This is commonly adopted given it reduces the amount of handling you need to do in your code for a given environment.

So when you are developing, `config.mongodb` points to a DB in your machine. In production, `config.mongodb` points to a production DB. It is the same thing for tests, when `NODE_ENV === 'test'`, `config.mongodb` will point to a DB used for tests. 

So when you said before that this will be stored in config:
```
exports.mongodb = "mongodb://brainlife_mongodb/auth";
exports.mongodb_test = 'mongodb://brainlife_mongodb/auth_test';
```

what I mean is that the config should be:
```
exports.mongodb = process.env.NODE_ENV === 'test' ? "mongodb://brainlife_mongodb/auth_test" : "mongodb://brainlife_mongodb/auth";
```

This change will abstract away the decision of "which DB should it use" to the configuration, and not to the code.